### DISCUSSION TITLE
Multi-Task Classification is not implemented

### DISCUSSION THREAD

**Discussion by fabiogeraci** [manujosephv/pytorch_tabular#49] [General]
Thank you very much for the great work. do you have any plans to implement Multi-Task Classification

**Comment by IvoryLu** [comment]
Thank you for the fantastic job. If I'm going to implement Multi-Task Classification myself, any recommendations on where to start from?

**Reply by manujosephv** [reply]
@IvoryLu Sorry for the very very late reply... So, the only thing which is stopping multi-task classification is the data-pipeline and some internal configs. The best place to start from will be to understand TabularDataModule and how it fits with TabularDatamodel and the different configs.
